### PR TITLE
RET-1838

### PR DESCRIPTION
**Pull request by cindychmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RET-1838

### Change description ###

Update for handle 0 to 6 number of Acas certs

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/Et1VettingService.java [hunk 1]
```diff
@@ -10,38 +10,52 @@ public class Et1VettingService {
...
+    static final String BEFORE_LABEL_ACAS_OPEN_TAB = "<br/><a target=\"_blank\" href=\"%s\">"
+            + "Open the Documents tab to view/open Acas certificates (opens in new tab)</a>";
 
     /**
      * Update et1VettingBeforeYouStart.
      * @param caseDetails Get caseId and Update caseData
      */
-    public void initialBeforeYouStart(CaseDetails caseDetails) {
+    public void initialiseEt1Vetting(CaseDetails caseDetails) {
+        caseDetails.getCaseData().setEt1VettingBeforeYouStart(initialBeforeYouStart(caseDetails));
```
Comment (robin-ieong) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/Et1VettingService.java:27]: I don't think this method is required, why not just setEt1VettingBeforeYouStart directly from the controller method?

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/Et1VettingServiceTest.java [hunk 2]
```diff
@@ -31,47 +34,85 @@ void setUp() {
...
+        var acasBinaryUrl5 = "/documents/acas5555-4ef8ca1e3-8c60-d3d78808dca1/binary";
+
+        List<DocumentTypeItem> documentTypeItemList = new ArrayList<>();
+        documentTypeItemList.add(createDocumentTypeItem(ET1_DOC_TYPE, et1BinaryUrl1));
+        documentTypeItemList.add(createDocumentTypeItem(ACAS_DOC_TYPE, acasBinaryUrl1));
+        documentTypeItemList.add(createDocumentTypeItem(ACAS_DOC_TYPE, acasBinaryUrl2));
+        documentTypeItemList.add(createDocumentTypeItem(ACAS_DOC_TYPE, acasBinaryUrl3));
+        documentTypeItemList.add(createDocumentTypeItem(ACAS_DOC_TYPE, acasBinaryUrl4));
+        documentTypeItemList.add(createDocumentTypeItem(ACAS_DOC_TYPE, acasBinaryUrl5));
+        caseDetails.getCaseData().setDocumentCollection(documentTypeItemList);
```
Comment (robin-ieong) [inline src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/Et1VettingServiceTest.java:64]: Can we do the above lines in the @BeforeEach method? Same applies to the next method, that should be able to reduce  duplicated code.

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/Et1VettingService.java [hunk 3]
```diff
@@ -10,38 +10,49 @@ public class Et1VettingService {
...
+    }
+
+    public String initialBeforeYouStart(CaseDetails caseDetails) {
```
Comment (hsjhita) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/Et1VettingService.java:30]: private

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/Et1VettingService.java [hunk 4]
```diff
@@ -10,38 +10,49 @@ public class Et1VettingService {
...
 
-        var et1BinaryUrl = createDocLinkDefault(caseDetails.getCaseId());
-        var acasBinaryUrl = createDocLinkDefault(caseDetails.getCaseId());
+        String et1Display = "";
+        StringBuilder acasDisplayStringBuilder = new StringBuilder();
+        int acasCount = 0;
 
         var documentCollection = caseDetails.getCaseData().getDocumentCollection();
         if (documentCollection != null && !documentCollection.isEmpty()) {
```
Comment (hsjhita) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/Et1VettingService.java:37]: CollectionUtil.isNotEmpty

**Review by nickwest1 — CHANGES_REQUESTED** [review]
We may need to have a discussion on these comments Cindy.  Possibly a pairing session.

**Review by robin-ieong — APPROVED** [review]

**Pull request merged**